tests/kafkatest/services/security/security_config.py

```diff
@@ -120,13 +123,15 @@ def setup_node(self, node):
 
         if self.has_sasl:
             node.account.ssh("mkdir -p %s" % SecurityConfig.CONFIG_DIR, allow_fail=False)
-            jaas_conf_file = self.sasl_mechanism.lower() + "_jaas.conf"
+            jaas_conf_file = "jaas.conf"
```

Comment:
Any reason this changed? I'm not sure its an issue, but it seems like we may have named the file with the SASL mechanism previously to at least aid in debugging.